tasks.star

```diff
@@ -358,22 +359,93 @@ def configure():
         ],
     )
 
+    db_network = "nebula"
+    db_container = "nebula-db"
+    db_port = 1234
```

Comment:
This should probably be an `option()`. The other two variables as well.

Reply:
Done! Changed the default port as well.